src/Service/RedisReader.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace App\Service;
+
+use Predis\Client;
+
+/**
+ * Class RedisReader
+ * @package App\Service
+ *
+ * @author Roberto Umbertini <daeronmalnwe@gmail.com>
+ */
+class RedisReader implements ReaderInterface
```

Comment:
also this class is yet present here `Stats/Reader/RedisReader.php` by @garak!

Reply:
@daeronmalnwe again, can you take a look? :)